probe-rs/src/flashing/flash_algorithm.rs

```diff
@@ -1,5 +1,6 @@
+use probe_rs_target::{FlashProperties, PageInfo, RamRegion, RawFlashAlgorithm, SectorInfo};
```

Comment:
Are the changes in this file related to the sequences at all?

Reply:
I think this is something that came in because I merged and didn't rebase.